vm_core.h

```diff
@@ -230,6 +230,8 @@ union ic_serial_entry {
     VALUE data[2];
 };
 
+#define IMEMO_CONST_CACHE_SHAREABLE IMEMO_FL_USER0
```

Comment:
Could you add a comment to explain what this flag means? (single constant is ractor-shareable?)